EasyLearn/.gitignore

```diff
@@ -35,3 +35,6 @@
 /config/master.key
 
 /config/database.yml
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
When you include any important Rails system files, it is essential to include an example file that can be used by anyone who downloads your project. And indicate it in the README.md, for better file management.

Reply:
Done.